doc/Rules.md

```diff
@@ -845,26 +845,26 @@ heading name; headings with the same content can cause problems with that.
 
 <a name="md025"></a>
 
-## MD025 - Multiple top level headings in the same document
+## MD025 - Multiple top-level headings in the same document
 
 Tags: headings, headers
 
 Aliases: single-title, single-h1
 
 Parameters: level, front_matter_title (number; default 1, string; default "^\s*"?title"?\s*[:=]")
 
-This rule is triggered when a top level heading is in use (the first line of
-the file is a h1 heading), and more than one h1 heading is in use in the
+This rule is triggered when a top-level heading is in use (the first line of
+the file is an h1 heading), and more than one h1 heading is in use in the
 document:
 
 ```markdown
 # Top level heading
 
-# Another top level heading
+# Another top-level heading
 ```
 
 To fix, structure your document so that there is a single h1 heading that is
-the title for the document, and all later headings are h2 or lower level
+the title for the document and all later headings are h2 or lower level
```

Comment:
IMO, removal of this comma makes the sentence harder to read.

Reply:
What about:
> To fix, structure your document so there is a single h1 heading that is the title for the document. Subsequent headings must be lower-level headings h2, h3, etc.:

Reply:
Yep, breaking into two sentences seems better here.